src/ontology/Makefile

```diff
@@ -264,6 +266,43 @@ $(ONT)-with-equivalents.obo: $(ONT)-with-equivalents.owl
 $(ONT)-with-equivalents.json: $(ONT)-with-equivalents.owl
 	$(ROBOT) convert -i $< -o $@.tmp.json && mv $@.tmp.json $@
 
+ONTOLOGYTERMS=$(TMPDIR)/mondo_classes.txt
+
+$(ONTOLOGYTERMS): reasoned.owl
+	$(ROBOT) query -f csv -i $< --query ../sparql/signature/mondo-classes.sparql $@
+
+SIMPLESEED=$(TMPDIR)/simple_seed.txt
+
+$(SIMPLESEED): reasoned.owl $(ONTOLOGYTERMS)
+	$(ROBOT) query -f csv -i $< --query ../sparql/signature/simple-seed.sparql $@.tmp &&\
+	cat $@.tmp $(ONTOLOGYTERMS) | sort | uniq >  $@ &&\
+	echo "http://www.geneontology.org/formats/oboInOwl#SubsetProperty" >> $@ &&\
+	echo "http://www.geneontology.org/formats/oboInOwl#SynonymTypeProperty" >> $@
+
+# foo-simple: (edit->reason,relax,reduce,drop imports, drop every axiom which contains an entity outside the "namespaces of interest")
+# drop every axiom: filter --term-file keep_terms.txt --trim true
+#	remove --select imports --trim false
+# NOTE FOR FUTURE: this is slightly different from the usual simple as it completely removes all
+# Object properties from the ontology (remove --select object-properties).
+$(ONT)-simple.owl: reasoned.owl $(OTHER_SRC) $(SIMPLESEED) $(IMPORT_FILES)
+	$(ROBOT) merge --input $< \
+		reason --equivalent-classes-allowed none --exclude-tautologies true --annotate-inferred-axioms true \
+		relax \
+		remove --axioms equivalent \
+		remove --select object-properties \
```

Comment:
why are object properties being removed for "simple"?

Reply:
This was the decision we made: the simple release product should only be the class hierarchy, nothing else. In particular: no relationships, such as disease characteristics and similar.